### PR TITLE
Fail2ban: v0.9.6

### PR DESCRIPTION
**Pull request by andrewying**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

Upgrade fail2ban to v0.9.6

### CODE REVIEW

**Comment by andrewying** [comment]
Hey! Just wondering if there'll be any updates - is any changes needed?

File: Formula/fail2ban.rb [hunk 1]
```diff
@@ -13,27 +13,23 @@ class Fail2ban < Formula
     sha256 "f39d0f4aa122b1e40ce05ad9010901beefacd560c5d84960eed4448daa3915f2" => :mavericks
   end
 
+  depends_on :python
```
Comment (fxcoudert) [inline Formula/fail2ban.rb]: This should probably be `depends_on :python if MacOS.version <= :snow_leopard`

File: Formula/fail2ban.rb [hunk 2]
```diff
@@ -76,4 +72,9 @@ def plist; <<-EOS.undent
       </plist>
     EOS
   end
+
+  test do
+    system bin/"fail2ban-client", "-h"
```
Comment (fxcoudert) [inline Formula/fail2ban.rb]: Could we do a better test?

**Review by fxcoudert — CHANGES_REQUESTED** [review]
The `caveats` talks about Mountain Lion, and macOS 10.4 and 10.5. Please link to updated doc

**Pull request closed without merging**